modules/ROOT/pages/georeplication.adoc

```diff
@@ -0,0 +1,97 @@
+= Georeplication
```

Comment:
General comment: this document is pretty much the same as OSS Pulsar doc ([link](https://pulsar.apache.org/docs/en/administration-geo/#connect-replication-clusters)).

I actually want to change this doc to be a broader scope including topics like below:

Topic: Multi region/data-center Pulsar deployment
1) Overview of local and remote message replication in Pulsar
2) Multi-region/data-center Pulsar deployment pattern: synchronous vs asynchronous "geo-" replication
3) Synchronous "geo-" replication: region awareness and rack awareness
4) Asynchronous "geo-" replication: with or without Global Zookeper
5) Message processing semantics with "geo-" replication

Reply:
The OSS Pulsar geo-replication doc is clear enough and I don't think we need to repeat them. We can simply point to the OSS doc.

For the above proposed scope change, let's create a new ticket.